javaslang/generator/Generator.scala

```diff
@@ -51,6 +51,377 @@ def generateMainClasses(): Unit = {
+           */
+          public static <T> $FutureType<T> future(T result) {
+            return $FutureType.successful(result);
+          }
+
+          /$javadoc
+           * Alias for {@link $FutureType#successful($ExecutorServiceType, Object)}
+           *
+           * @param <T>             The value type of a successful result.
+           * @param executorService An {@code ExecutorService}.
+           * @param result          The result.
+           * @return A succeeded {@code Future}.
+           * @throws NullPointerException if executorService is null
+           */
+          public static <T> $FutureType<T> future($ExecutorServiceType executorService, T result) {
+            return $FutureType.successful(executorService, result);
+          }
+
+          /$javadoc
+           * Alias for {@link $FutureType#failed(Throwable)}
+           *
+           * @param <T>       The value type of a successful result.
+           * @param exception The reason why it failed.
+           * @return A failed {@code Future}.
+           * @throws NullPointerException if exception is null
+           */
+          public static <T> $FutureType<T> future(Throwable exception) {
+            return $FutureType.failed(exception);
+          }
+
+          /$javadoc
+           * Alias for {@link $FutureType#failed($ExecutorServiceType, Throwable)}
+           *
+           * @param <T>             The value type of a successful result.
+           * @param executorService An executor service.
+           * @param exception       The reason why it failed.
+           * @return A failed {@code Future}.
+           * @throws NullPointerException if executorService or exception is null
+           */
+          public static <T> $FutureType<T> future($ExecutorServiceType executorService, Throwable exception) {
+            return $FutureType.failed(executorService, exception);
+          }
+
+          /$javadoc
+           * Alias for {@link Lazy#of($SupplierType)}
+           *
+           * @param <T>      type of the lazy value
+           * @param supplier A supplier
+           * @return A new instance of Lazy
+           */
+          public static <T> Lazy<T> lazy($SupplierType<? extends T> supplier) {
+            return Lazy.of(supplier);
+          }
+
+          /$javadoc
+           * Alias for {@link Option#of(Object)}
+           *
+           * @param <T>   type of the value
+           * @param value A value
+           * @return {@code Some(value)} if value is not {@code null}, {@code None} otherwise
+           */
+          public static <T> $OptionType<T> option(T value) {
+            return $OptionType.of(value);
+          }
+
+          /$javadoc
+           * Alias for {@link Option#some(Object)}
+           *
+           * @param <T>   type of the value
+           * @param value A value
+           * @return {@code Some(value)}
+           */
+          public static <T> $OptionType<T> some(T value) {
```

Comment:
I think - yes we need aliases and for this classes to: user can add one import and obtain access to most useful functions